.github/workflows/sign-example.yml

```
@@ -26,6 +26,7 @@ jobs:
     uses: sillsdev/codesign/.github/workflows/sign.yml@v2
```

I think these tags should all be the same number across the examples, would you agree?
```suggestion
    uses: sillsdev/codesign/.github/workflows/sign.yml@v2.1
```